### PR TITLE
Add new widget for RescueTime

### PR DESCRIPTION
**Pull request by wozboz**
[wozboz](https://github.com/wozboz) ✨ Feature Medium [wozboz /master → Lissy93/dashy](https://github.com/Lissy93/dashy/tree/master) Commits: 4 | Files Changed: 4 | Additions: 144 [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=Lissy93&utm_campaign=badge_info)

*Thank you for contributing to Dashy! So that your PR can be handled effectively, please populate the following fields (delete sections that are not applicable)*

**Category**: 
> One of: Feature

**Overview**
> Added a widget for the time tracking app "RescueTime". The widget shows how much time you have spent on each category (e.g. Business, Programming, ...)

**Screenshot** _(if applicable)_

**Code Quality Checklist** _(Please complete)_
- [ x ] All changes are backwards compatible
- [ x ] All lint checks and tests are passing
- [ x ] There are no (new) build warnings or errors

### CODE REVIEW

File: docs/widgets.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Dashy has support for displaying dynamic content in the form of widgets. There a
   - [Mvg Departure](#mvg-departure)
   - [Mvg Connection](#mvg-connection)
   - [Custom search](#custom-search)
+  - [Rescuetime overview] (#rescuetime-overview)
```
Comment (Lissy93) [inline docs/widgets.md:42]: There's a space between the name and link which makes it not [render](https://github.com/Lissy93/dashy/blob/be0f51d46a3b0e57488f72faa43c4752c3a6cadb/docs/widgets.md) quite right.
```diff
- - [Rescuetime overview] (#rescuetime-overview)
                         ^
                         ╰ Just remove this ;)
+ - [Rescuetime overview](#rescuetime-overview)                       
```

File: src/components/Widgets/RescueTime.vue [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        day = `0${day}`;
+      }
+      if (month < 10) {
+        month = `0${month}`;
+      }
+      return `${day}-${month}-${year}`;
+    },
+    calculateTimeCategories(timeArray) {
+      const results = {};
+      // console.log(timeArray.rows[0]);
```
Comment (Lissy93) [inline src/components/Widgets/RescueTime.vue:68]: Forgot to remove this?
And the `// import axios from 'axios';` on L21 ;)

**Review by Lissy93 — APPROVED** [review]
Looks great :)

File: src/components/Widgets/RescueTime.vue (hunk 2 above)
Comment (wozboz) [reply src/components/Widgets/RescueTime.vue]: Yep... looked over this about 10 times, somehow missed it. Thanks for removing it :)

**Pull request merged**